react-project/src/components/Empty/Empty.tsx

```diff
@@ -5,16 +5,16 @@ type EmptyProps = {
   imageName: string;
 };
 
-function Empty(props: EmptyProps) {
+function Empty({ text, imageName }: EmptyProps) {
   return (
     <div className="empty-wrapper">
       <div
         className="empty"
         style={{
-          backgroundImage: `url(./src/assets/${props.imageName}.jpg)`,
+          backgroundImage: `url(./src/assets/${imageName}.jpg)`,
         }}
```

Comment:
It won't be work on production...
You should move assets to public (main project folder) and make:
backgroundImage: \`url(/assets/${imageName}.jpg)\`,